<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDocDialog</class>
 <widget class="QDialog" name="ImportDocDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Import</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="fileSelDir">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="fileSelBtn">
     <property name="text">
      <string>&amp;Select...</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="2" column="0">
       <widget class="QTableView" name="previewView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::NoPen</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="previewBtn">
        <property name="text">
         <string>PreView Files</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="delBtn">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="whatsThis">
      <string/>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Import Location:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>&amp;Path to Import Into:</string>
     </property>
     <property name="buddy">
      <cstring>fileTypeCmb</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="destDir">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;File Type:</string>
     </property>
     <property name="buddy">
      <cstring>fileSelDir</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="fileTypeCmb">
     <item>
      <property name="text">
       <string>All files(*.*)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All supported files</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Office files(*.doc,*.docx,*.xls,*.xlsx,*.ppt,*.pptx)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pdf files(*.pdf)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Html files(*.html,*.htm)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Picture file(*.jpeg,*.jpg,*.png,*.bmp,*.gif)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flash files(*.flv,*.swf)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Source files(*.cpp,*.h,*.hpp,*.c,*.java,*.js,*.cs,*.vbs,*.sql,*.rb,*.py,*.asp,*.aspx,*.jsp,*.php,*.xml)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Txt files(*.txt,*.ini,*.bar,*.inf)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Movie fies(*.rm,*.rmvb,*.avi,*.mpg,*.mlv,*.mpe,*.mpeg,*.m2v)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="customFileType">
     <property name="toolTip">
      <string>eg: *.bsp,*.axs</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="inclueSubDirCheck">
     <property name="toolTip">
      <string>Switch it on if the header should not be imported</string>
     </property>
     <property name="text">
      <string>Include Sub Directory </string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="excludefileType">
     <property name="toolTip">
      <string>eg: *.bsp,*.axs</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Exclude File Type:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Custom File Type:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportDocDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
